Blog_part_2.md

```diff
@@ -0,0 +1,324 @@
+
+    - name: TARGET_HOST
+      key: TARGET_HOST
+      value: http://EXTERNAL-IP:8080
+
+#### Configure the Controller Docker Image
+
+Now we need to tag our existing Locust Docker Image so we can deploy it to Google Image Repository. If you do not already have an image created please go to the top of this guide and follow those directions. Also ensure that you have a Google Cloud Project available to use. If you do not follow the "Before you begin" instructions from the [Google Container Engine Quick Start](https://cloud.google.com/container-engine/docs/quickstart). Once you have a Google Cloud Project setup replace PROJECT-ID throughout the rest of this guide with your Google Cloud Project Id.
+
+    $ docker tag locust-tasks gcr.io/PROJECT-ID/locust-tasks
+    $ gcloud docker -- push gcr.io/PROJECT-ID/locust-tasks
+
+**Note:** you are not required to use the Google Container Registry. If you'd like to publish your images to the [Docker Hub](https://hub.docker.com) please refer to the steps in [Working with Docker Hub](https://docs.docker.com/userguide/dockerrepos/).
+
+Once the Docker image has been rebuilt and uploaded to the registry you will need to edit the controllers with your new image location. Specifically, the `spec.template.spec.containers.image` field in each controller controls which Docker image to use.
+
+If you uploaded your Docker image to the Google Container Registry:
+
+    image: gcr.io/PROJECT-ID/locust-tasks:latest
+
+If you uploaded your Docker image to the Docker Hub:
+
+    image: USERNAME/locust-tasks:latest
+
+**Note:** the image location includes the `latest` tag so that the image is pulled down every time a new Pod is launched. To use a Kubernetes-cached copy of the image, remove `:latest` from the image location.
+
+#### Deploy Kubernetes Cluster
+
+First create the [Google Container Engine](http://cloud.google.com/container-engine) cluster using the `gcloud` command as shown below. 
+
+**Note:** This command defaults to creating a three node Kubernetes cluster (not counting the master) using the `n1-standard-1` machine type. Refer to the [`gcloud container clusters create`](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create) documentation information on specifying a different cluster configuration.
+
+    $ gcloud container clusters create CLUSTER-NAME
+
+If you do not have a default Google Cloud Project Id set you can append the `--project=` argument to all of your gcloud commands like so:
+
+    $ gcloud container clusters create CLUSTER-NAME --project=PROJECT-ID
+
+If you do not know if you have a project set run the following command:
+
+    $ glcoud config list
+
+And you should see an output like so:
+
+    [compute]
+    zone = us-central1-a
+    [core]
+    account = email.address@somedomain.com
+    project = your-google-cloud-project-id
+
+I recommend setting both your project and compute zone. You can find the directions for both in the Google Cloud Tools Documentation.
```

Comment:
I think we should add a link to this - perhaps https://cloud.google.com/compute/docs/regions-zones/changing-default-zone-region